presets/4.3/rates/Garban_race.txt

```diff
@@ -0,0 +1,30 @@
+#$ TITLE: Garban Official race rates
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: racing, rates, DDC, Acrobaix, Garban
+#$ AUTHOR: Garban
+#$ DESCRIPTION: Racing rates from Josep de la Fuente (Garban), one of the fastest pilot in Catalonia.
+#$ DESCRIPTION: With 482 deg/sec on roll, pitch and yaw 
+#$ DESCRIPTION: Also with a throttle curve in order to improve low throttle 
```

Comment:
"Josep's throttle curve is available as an option here. Check it out!"

Reply:
Done!